.vscode/settings.json

```diff
@@ -0,0 +1,7 @@
+{
+  "workbench.colorCustomizations": {
+    "activityBar.background": "#083509",
+    "titleBar.activeBackground": "#0C4B0C",
+    "titleBar.activeForeground": "#F4FDF5"
+  }
+}
```

Comment:
you may want to add `.vscode` to `.gitignore`

Reply:
definitly